go.mod

```diff
@@ -13,7 +13,7 @@ require (
 	github.com/dell/gofsutil v1.8.0
 	github.com/dell/goiscsi v1.3.0
 	github.com/dell/gonvme v1.0.0
-	github.com/dell/gopowerstore v1.7.1-0.20220401105348-104d2bf041c3
+	github.com/dell/gopowerstore v1.7.1-0.20220420052716-a1dd228e16eb
```

Comment:
This will change upon merging gopowerstore to `main`

Reply:
Yes, I just updated this file for now, for fixing compilation issues in gosec and golint checks. I shall update it once gopowerstore PR is merged to main.